.github/workflows/pr_check.yml

```
@@ -0,0 +1,67 @@
+          - python-version: "3.10"
+            tox-env-py: "310"
+          - os: "ubuntu-20.04"
+            tox-env-os: "lin"
+          - os: "windows-2022"
+            tox-env-os: "win"
+    name: pr test (${{ matrix.os }}, Python ${{ matrix.python-version }})
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Installing Rust toolchain
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.4)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)